### DISCUSSION TITLE
templater and

### DISCUSSION THREAD

**Discussion by Belliar** [BryanJenksCommunity/FAQ#165] [Q&A] [answered]
Hi Bryan

Thank you for all the content you have published, I have seen most of your content, loved the 4h video on your workflow, still studying it!

Anyways, I see that you Included in your templates the following commands, could you please let me know if you have released some content where you explain in detail how you use this? I am currently going through my templates, and it seems like this would be extremely helpful, but i have not figured out how to use this command (TP.FILE.CURSOR), it seems like its some sort of macro that you created?

- `Tags:` <%tp.file.cursor(1)%>
- `URL:` <<%tp.file.cursor(2)%>>
- `Channel/Host:` <%tp.file.cursor(3)%>
- `Reference:` 
- `Publish Date:` <%tp.file.cursor(4)%>

Happy New Year!

Thank you
Matt

**Comment by tallguyjenks** [comment, marked as answer]
the cursors are activated via templater, im not sure if ive covered it or not, i may have in my templater video but its to jump the cursor to different locations
